beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -610,12 +613,13 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
                         active_batches.remove(&id);
                     }
                 }
-                BatchState::Failed | BatchState::Poisoned | BatchState::AwaitingDownload => {
-                    unreachable!("batch indicates inconsistent chain state while advancing chain")
-                }
+                BatchState::Failed | BatchState::Poisoned | BatchState::AwaitingDownload => debug!(
```

Comment:
similarly here, if it was unreachable, i'd expect this to be a crit right? 
Or this this likely to happen?

Reply:
no, this is a CRIT too